website/src/content/docs/guides/index.md

```diff
@@ -35,3 +35,106 @@ Install the following charts if not already installed:
 - [Snapshot-Controller](https://truecharts.org/charts/system/snapshot-controller/): Required for Volsync
 - [Volsync](https://truecharts.org/charts/system/volsync/): For Backup and Restore of PVCs
 - [Traefik-CRDS](https://truecharts.org/charts/system/traefik-crds/) & [Traefik](https://traefik.io/traefik/): For Ingress and Reverse Proxying
+
+## Upstream Operators
+
+Truecharts relies on multiple Charts for functionality like Postgres Databases and Metrics.
+Therefore we require certain Charts to be installed. Below you will find example configurations for most of them:
+
+### Cert-Manager
+
+Cert-Manager is used  together with our clusterissuer to create SSL certificates for ingress.
+The chart installation can be found [here](https://cert-manager.io/docs/installation/helm/).
+
+Example configuration:
+
+```yaml
+
+crds:
+  enabled: true
+dns01RecursiveNameservers: "1.1.1.1:53,1.0.0.1:53"
+dns01RecursiveNameserversOnly: false
+enableCertificateOwnerRef: true
+
+```
+
+### Cloudnative-PG
+
+Cloudnative-PG is used for Postgres databases in many of our charts.
+The chart can be found [here](https://cloudnative-pg.io/charts/).
+
+Example configuration:
+
+```yaml
+
+crds:
+  create: true
+
+```
+
+### Prometheus
+
+Kube-promotheus-stack is used for metrics. 
+The chart can be found [here](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack).
+
+As we provide our own grafana with included dashboard. We recommend to disable grafana in the kube-prometheus-stack:
+
+```yaml
+grafana:
+  enabled: false
+  forceDeployDashboards: true
+  defaultDashboardsEnabled: true
+```
+
+We generally advice to run the full kube-prometheus-stack but as it is quite resource intensive you can run the minimum requirement which only requires to add the CRDs and Operator. This can be done like this:
```

Comment:
```suggestion
We generally advice to run the full kube-prometheus-stack but as it is quite resource intensive you can run the minimum requirement which only requires to add the CRDs. This can be done like this:
```